api/lib/domain/read-models/CertifiedBadgeImage.js

```diff
@@ -30,43 +30,43 @@ class CertifiedBadgeImage {
       });
     }
 
-    if (partnerKey === PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_AUTONOME) {
+    if (partnerKey === PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_INITIE) {
       return new CertifiedBadgeImage({
-        path: 'https://images.pix.fr/badges-certifies/pix-edu/autonome.svg',
+        path: 'https://images.pix.fr/badges/Pix_plus_Edu-1-Initie-certif.svg',
         isTemporaryBadge: true,
-        levelName: 'Autonome',
+        levelName: 'Initié (entrée dans le métier)',
       });
     }
 
-    if (partnerKey === PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_AVANCE) {
+    if (partnerKey === PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_CONFIRME) {
```

Comment:
Question,  il y a t'il une raison de ne pas regrouper les 2?
```suggestion
    if (partnerKey === PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_CONFIRME || partnerKey === PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_CONFIRME) {
```

Reply:
Done